.github/workflows/chromatic.yml

```diff
@@ -0,0 +1,28 @@
+# .github/workflows/chromatic.yml
+
+# Workflow name
+name: 'Chromatic'
+
+# Event for the workflow
+on: push
```

Comment:
Are you sure you want to run this on each push? Can we only run on each merge to develop

Also can you create another PR to deploy storybook in github pages

Reply:
- I was thinking it would be good to have preview for each branch while are reviewing it so that its easier to spot out changes for storybook and for non technical people also.

- Hmm not sure if deploying to github pages is best option since you already have permalink for each branch im just not sure how to implement this so that you can see deployment like for page preview?
-  Here is example for this branch https://feature-1088-story-book--62b3528bdaa8110997aa1b57.chromatic.com https://www.chromatic.com/docs/permalinks